mapper/mapper.go

```diff
@@ -276,9 +276,13 @@ func CreateVersionsList(ctx context.Context, d dataset.Model, edition dataset.Ed
 
 		var correctionReasons []string
 		const correctionAlertType = "correction"
-		for _, alert := range *ver.Alerts {
-			if alert.Type == correctionAlertType {
-				correctionReasons = append(correctionReasons, alert.Description)
+		if *ver.Alerts != nil {
+			for _, alert := range *ver.Alerts {
+				if &alert != nil {
+					if alert.Type == correctionAlertType {
```

Comment:
It's more common in Go to collapse these 2 conditions into a single if statement to reduce the indentation as they have no elses to require the separation

Reply:
Adjusted :)